messenger.go

```diff
@@ -264,6 +264,22 @@ func (m *Messenger) Response(to int64) *Response {
 	}
 }
 
+// Send func will send a message to a user without first receiving a message.
+func (m *Messenger) Send(to Recipient, message string, replies []QuickReply) error {
```

Comment:
For the sake of keeping the API consistent, could we rename this function to `SendWithReplies` and then create another function `Send(to Recipient, message string)` (Notice the lack of `replies []QuickReply`).

It'd probably look something like this (untested):

```go
func (m *Messenger) Send(to Recipient, message string) error {
    return m.SendWithReplies(to, message)
}
```